src/main/java/org/javarosa/core/model/FormDef.java

```diff
@@ -1362,50 +1357,14 @@ public void populateDynamicChoices(ItemsetBinding itemset, TreeReference curQRef
         itemset.setChoices(choices, this.getLocalizer());
     }
 
-    public QuestionPreloader getPreloader() {
-        return preloader;
-    }
-
-    public void setPreloader(QuestionPreloader preloads) {
-        this.preloader = preloads;
-    }
-
     public String toString() {
         return getTitle();
     }
 
-    /**
-     * Preload the Data Model with the preload values that are enumerated in the
-     * data bindings.
-     */
-    public void preloadInstance(TreeElement node) {
-        // if (node.isLeaf()) {
-        IAnswerData preload = null;
-        if (node.getPreloadHandler() != null) {
-            preload = preloader.getQuestionPreload(node.getPreloadHandler(),
-                    node.getPreloadParams());
-        }
-        if (preload != null) { // what if we want to wipe out a value in the
-            // instance?
-            node.setAnswer(preload);
-        }
-        // } else {
-        if (!node.isLeaf()) {
-            for (int i = 0; i < node.getNumChildren(); i++) {
-                TreeElement child = node.getChildAt(i);
-                if (child.getMult() != TreeReference.INDEX_TEMPLATE)
-                    // don't preload templates; new repeats are preloaded as they're created
-                    preloadInstance(child);
-            }
-        }
-        // }
-    }
-
-    public boolean postProcessInstance() {
+    public void postProcessInstance() {
         if(!isCompletedInstance) {
             actionController.triggerActionsFromEvent(Action.EVENT_XFORMS_REVALIDATE, this);
         }
-        return postProcessInstance(mainInstance.getRoot());
```

Comment:
thanks, I actually should have removed the (now unused) method too. All this function was doing was triggering post processing for preloaders which are now gone so, unneeded.